### PR TITLE
Bug 1346155 - Tests for gmw.cn, r=Gijs

### PR DESCRIPTION
**Pull request by evanxd**
Tests for gmw.cn(global rank 89). The test page is http://tech.gmw.cn/2017-03/10/content_23937776.htm.

### CODE REVIEW

File: test/test-pages/gmw/expected-metadata.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "title": "宇航员在太空中喝酒会怎么样？后果很严重 _探索者 _光明网",
```
Comment (evanxd) [inline test/test-pages/gmw/expected-metadata.json]: Filed a follow issue #366 to strip "_探索者 _光明网" at end of the title.

**Comment by evanxd** [comment]
Hi Gijs,

Could you help review the test? Thanks.

**Comment by gijsk** [comment]
Looks like this:

> 　　出品︱网易科学人栏目组 胖胖
>　　作者︱春春 [返回光明网首页]
> [责任编辑:肖春芳]

should ideally be removed, but there's no real markup to go on?

Also, the empty `<ins>` element should be removed, but I think we have an issue on file for this already. otherwise this looks fine.

**Comment by evanxd** [comment]
```
　　出品︱网易科学人栏目组 胖胖
　　作者︱春春 [返回光明网首页]
[责任编辑:肖春芳]
```
It's hard to say this is a part of the article, or not. I would say we don't need to remove it.

**Pull request merged**